src/Goerli-DssSpell.t.base.sol

```diff
@@ -962,10 +991,43 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
         assertEq(vestDai.rxd(_index), _claimed,    "rxd");
     }
 
-    function getMat(bytes32 _ilk) internal view returns (uint256 mat) {
+    function getIlkMat(bytes32 _ilk) internal view returns (uint256 mat) {
         (, mat) = spotter.ilks(_ilk);
     }
 
+    function getIlkDuty(bytes32 _ilk) internal view returns (uint256 duty) {
+        (duty,)  = jug.ilks(_ilk);
+    }
+
+    function setIlkMat(bytes32 ilk, uint256 amount) internal {
+        hevm.store(
+            address(spotter),
+            bytes32(uint256(keccak256(abi.encode(ilk, uint256(1)))) + 1),
+            bytes32(amount)
+        );
+        assertEq(getIlkMat(ilk), amount, concat("TestError/setIlkMat-", ilk));
+    }
+
+    function setIlkRate(bytes32 ilk, uint256 amount) internal {
+        hevm.store(
+            address(vat),
+            bytes32(uint256(keccak256(abi.encode(ilk, uint256(2)))) + 1),
+            bytes32(amount)
+        );
+        (,uint256 rate,,,) = vat.ilks(ilk);
+        assertEq(rate, amount, concat("TestError/setIlkRate-", ilk));
```

Comment:
Not using this, was it your intention to keep it around as a helper for later?

Reply:
storage slot looks right 👍

Reply:
Was going to use it and then went a different route. It probably wouldn't hurt to build more helpers out like this, esp. because they can solve some stack trace problems for us.